app/controllers/portal/client_logins_controller.rb

```diff
@@ -49,7 +48,7 @@ def check_verification_code
         return if redirect_locked_clients # check if any records are already locked
         if @records.present? # we have at least one match and none are locked
           DatadogApi.increment("#{self.controller_name}.verification_codes.right_code")
-          redirect_to self.class.to_path_helper(action: :edit, id: hashed_verification_code, **extra_path_params)
```

Comment:
i made this controller and i just want to say thanks i hate it

Reply:
*er, i mean i made the `state_file/intake_logins_controller.rb` along with these changes to the client logins controller